pkg/controlplane/instance.go

```diff
@@ -637,6 +638,8 @@ var (
 		flowcontrolv1beta1.SchemeGroupVersion.WithResource("prioritylevelconfigurations"), // remove in 1.26
 		flowcontrolv1beta2.SchemeGroupVersion.WithResource("flowschemas"),                 // remove in 1.29
 		flowcontrolv1beta2.SchemeGroupVersion.WithResource("prioritylevelconfigurations"), // remove in 1.29
+		flowcontrolv1beta3.SchemeGroupVersion.WithResource("flowschemas"),                 // remove in 1.29
+		flowcontrolv1beta3.SchemeGroupVersion.WithResource("prioritylevelconfigurations"), // remove in 1.29
```

Comment:
v1beta3 is to be removed in 1.32 (deprecated in 1.29, removed in 1.32)

Reply:
+1

Reply:
thanks, comments updated